xml/hwbp_registers.xml

```diff
@@ -1135,10 +1267,14 @@
             Data used together with \FcsrTextraThirtytwoMhselect.
         </field>
         <field name="mhselect" bits="25:23" access="WARL" reset="0">
-            0: Ignore \FcsrTextraThirtytwoMhvalue.
+            <value v="0" name="ignore">
+            Ignore \FcsrTextraThirtytwoMhvalue.
+            </value>
 
-            4: This trigger will only match if the low bits of
+            <value v="4" name="equal">
```

Comment:
All modes have some sort of equality comparison.  It's a matter of what values are being compared.  I would say name=mcontext on this one.

Reply:
We really need names for all 3 of the options, and it's a bit of a mis-mash. I'm going with mcontext, mcontext_select, and vmid_select for now.

Reply:
This isn't perfect but perfect is the enemy of good.